netket/operator/_kinetic.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional, Callable, Union, List
+
+from netket.utils.types import DType, PyTree, Array
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import ContinousOperator
+
+import jax
+import jax.numpy as jnp
+
+
+class KineticEnergy(ContinousOperator):
+    r"""Returns the local kinetic energy (hbar = 1)
+                :math:`E_{kin} = -1/2 ( \sum_i \frac{1}{m_i} (\log(\psi))'^2 + (\log(\psi))'' )`
```

Comment:
it does not return the local kinetic energy. You should say this is the kinectic energy operator

Reply:
hmm I think it does return the local kinetic energy. When you do this for a lot of samples and take the mean, then it is the kinetic energy, right?

Reply:
yes this seems like the local estimator of the kinetic energy

Reply:
No, I mean that `KineticOperator` is the kinetic operator `\hat{E}_{kin}` according to the formula you give. 
It does not return the local kinetic energy which i would interpret more as meaning the expectation value of the kin. energy.

Reply:
Also that is the class description, not the `__init__` description. So this does not `return` but `it is`